experimental/e2e/fixture/fixture_test.go

```diff
@@ -0,0 +1,260 @@
+package fixture_test
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/grafana/grafana-plugin-sdk-go/experimental/e2e/fixture"
+	"github.com/grafana/grafana-plugin-sdk-go/experimental/e2e/storage"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFixtureAdd(t *testing.T) {
+	t.Run("should add request and response to fixture store", func(t *testing.T) {
+		req, res := setupFixture()
+		defer req.Body.Close()
```

Comment:
Just curious, is this necessary?

Reply:
the `defer`? i think this was to fix a failure from `mage lint`, but i'll double check

Reply:
No sorry I should have been more specific - I meant closing the request body actually 🤔

Reply:
sorry, yeah that's what i meant too. i wasn't sure if you meant that line or the test in general

Reply:
removed in https://github.com/grafana/grafana-plugin-sdk-go/pull/467